eventuate-core/src/test/scala/com/rbmhtechnology/eventuate/EventsourcedViewSpec.scala

```diff
@@ -536,3 +536,44 @@ class EventsourcedViewSpec extends TestKit(ActorSystem("test")) with WordSpecLik
     }
   }
 }
+
+object EventsourcedViewReplaySpec {
```

Comment:
Should be called `EventsourcedViewReplayRetrySpec` because other replay tests run elsewhere.